pkg/acpi/rsdp_linux.go

```diff
@@ -0,0 +1,200 @@
+// Copyright 2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package acpi can find and parse the RSDP pointer and struct.
+package acpi
+
+import (
+	"bufio"
+	"encoding/binary"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/u-root/u-root/pkg/boot/ebda"
+	"github.com/u-root/u-root/pkg/memio"
+)
+
+var (
+	defaultRSDP = []byte("RSDP PTR U-ROOT\x02")
+)
+
+// RSDP is the v2 version of the ACPI RSDP struct.
+type RSDP struct {
```

Comment:
rsdp struct should not be in _linux file?

Reply:
yep. done.